linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/tests/chrony_d_one_server_misconfigured.fail.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# packages = chrony
+
+{{{ bash_package_remove("ntp") }}}
+
+# Remove all pool options
+sed -i "/^pool.*/d" {{{ chrony_d_path }}}/10-servers.conf
+
+if ! grep "^server.*maxpoll 10" {{{ chrony_d_path }}}/10-servers.conf ; then
```

Comment:
These configs seem platform specific. Maybe the tests should be made platform specific?